javascript/javascript1/week2/exercises.js

```diff
@@ -0,0 +1,133 @@
+// STEP 2 @sergii-borodin
+
+// Step 3: Smart-ease - Goes Global!
+
+// Flight booking fullname function
+
+function getFullName(firstname, surname) {
+  const fullName = firstname + " " + surname;
+  return console.log(fullName);
+}
+
+const fullname1 = getFullName("Benjamin", "Hughes");
+const fullname2 = getFullName("Richard", "Chaplin");
+
+// Formal fullname
+
+function getFullname(firstname, surname, useFormalName, gender) {
+  const fullName = firstname + " " + surname;
+
+  if (useFormalName && gender === "male") {
+    const formalMaleName = "Lord " + fullName;
+    console.log(formalMaleName);
+    return formalMaleName;
+  } else if (useFormalName && gender === "female") {
+    const formalFemaleName = "Lady " + fullName;
+    console.log(formalFemaleName);
+    return formalFemaleName;
+  } else if (!useFormalName && gender === "male") {
+    const formalMaleName = "Lord " + fullName;
+    console.log(formalMaleName);
+    return formalMaleName;
+  } else if (!useFormalName && gender === "female") {
+    const formalFemaleName = "Lady " + fullName;
+    console.log(formalFemaleName);
+    return formalFemaleName;
+  } else if (
+    useFormalName === null ||
+    gender !== "male" ||
+    gender !== "female"
+  ) {
+    return console.log(
+      "To choose gender and a way, how you want to be addressed(formally or not) is required!"
+    );
+  }
+}
```

Comment:
Look into what you are returning. Sometimes it is a string, other times it is a console.log. Best to be more consistent.

Also look using an else statement, instead of only if/else-if